testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/openshift/OpenShiftTokenReviewEndpointTest.java

```diff
@@ -220,13 +227,9 @@ public void emptyScope() {
         ClientRepresentation clientRep = testRealm().clients().findByClientId("test-app").get(0);
 
         List<String> scopes = new LinkedList<>();
-        for (ClientScopeRepresentation s : testRealm().clients().get(clientRep.getId()).getDefaultClientScopes()) {
```

Comment:
When refactoring these tests, use `ClientAttributeUpdater` instead of rewriting the try-finally. See https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/account/AccountFormServiceTest.java#L1082-L1087

Reply:
Ok, thanks for advice.